tests/test_train.py

```diff
@@ -1,3 +1,83 @@
 import pytest,fastai
+from utils.fakes import *
+from utils.text import *
 
-#I'm all empty, please fill me with tests about the functions in train.py
\ No newline at end of file
+## filename: test_train.py
+## tests functions in train.py
```

Comment:
Last note: please remove all useless blank lines to save vertical space (fastai style guide), thanks!